default.env

```diff
@@ -87,6 +87,22 @@ NB_API_WORKERS=1
 # The default is 8 and should be sufficient for a large team of moderators
 NB_API_THREADS=8
 
+# Use this *only* if hosting your RCON behind a reverse proxy or on subdomains. Otherwise
+# leave it empty.
+#
+# If hosting on multiple subdomains (e.g. rcon1.example.com, rcon2.example.com, etc),
+# cookies and therefore login tokens are not shared amongst the different subdomains. This
+# value tells the RCON server to append a domain to these cookies so they can be shared among
```

Comment:
> to append a domain to these cookies
That doesn't make much sense, does it? The Cookies always have a domain set, by default it's the very same domain where the website is hosted.

These settings are _overwriting_ this domain to the value you specify. This should be reflected in the comment. Please also add the documentation link from Django :)

https://docs.djangoproject.com/en/5.1/ref/settings/#std-setting-SESSION_COOKIE_DOMAIN
https://docs.djangoproject.com/en/5.1/ref/settings/#csrf-cookie-domain

Reply:
Agreed, and resolved for re-review.  Sorry for the phrasing, I assume I was insane at the time.

Reply:
> The Cookies always have a domain set, by default it's the very same domain where the website is hosted.

This actually doesn't seem to be true in my testing.  [W3 claims the domain specifier is optional](https://www.w3.org/Protocols/rfc2109/rfc2109) and Django, if no `SESSION_COOKIE_DOMAIN` is set, leaves the `domain` field empty; it only appears in the browser because the browser infers it should be set (for obvious security), so effectively you're still right and I'm being pedantic.  Just documenting for my own knowledge/pitfalls, the cookie will be assigned by the *client* in scenarios when it's empty on-wire, and both perform evaluations/sets.  I'm not a webdev at all so this is news to me and it's *very* likely I misunderstand something.